jujugui/static/gui/src/app/components/generic-button/test-generic-button.js

```diff
@@ -88,12 +88,14 @@ describe('GenericButton', function() {
     assert.equal(stopPropagation.callCount, 1);
   });
 
-  it('displays the provided title', function() {
+  it('displays the provided title and tooltip', function() {
     var output = jsTestUtils.shallowRender(
         <juju.components.GenericButton
-          title="My action" />);
+          title="My action"
+          tooltip="My tooltip" />);
     assert.deepEqual(output,
       <button className="button--neutral"
+        title="My tooltip"
```

Comment:
Well, no. But only because we already had a title prop and it was actually being used for the button's content. Ideally we'd come back later and rename the title prop to something else (content) and then tooltip could become title. I can open an issue so we don't lose track of it.